rm/rm-server/src/test/resources/functionaltests/config/rm-with-db.ini

```diff
@@ -19,4 +19,5 @@ pa.rm.select.node.dynamicity=10000
 pa.rm.node.source.ping.frequency=10000
 pa.rm.client.ping.frequency=10000
 pa.rm.select.script.timeout=10000
-pa.rm.topology.enabled=false
```

Comment:
The LocalSelectionTest does need the topology to be enabled. The JobLegacySchema test in the scheduler needs both the topology and the topology-distance in order to pass.